resources/ts/Components/Integrations/Detail/OrganizersDatalist.tsx

```diff
@@ -0,0 +1,152 @@
+import { Input } from "../../Input";
+import type { Props as FormElementProps } from "../../FormElement";
+import { FormElement } from "../../FormElement";
+import React, { useRef, useState } from "react";
+import type { UiTPASOrganizer } from "../../../types/UiTPASOrganizer";
+import { debounce } from "lodash";
+import { useTranslation } from "react-i18next";
+import { Alert } from "../../Alert";
+import { ButtonIcon } from "../../ButtonIcon";
+import { faTrash } from "@fortawesome/free-solid-svg-icons";
+
+type Props = {
+  onChange: (organizers: UiTPASOrganizer[]) => void;
+  value: UiTPASOrganizer[];
+} & Omit<FormElementProps, "onChange" | "component">;
+
+export const OrganizersDatalist = ({ onChange, value, ...props }: Props) => {
+  const { t } = useTranslation();
+  const [isSearchListVisible, setIsSearchListVisible] = useState(false);
+  const [organizerList, setOrganizerList] = useState<UiTPASOrganizer[]>([]);
+  const [organizerError, setOrganizerError] = useState(false);
+  const organizersInputRef = useRef<HTMLInputElement>(null);
+
+  const handleGetOrganizers = debounce(
+    async (e: React.ChangeEvent<HTMLInputElement>) => {
+      const response = await fetch(`/organizers?name=${e.target.value}`);
+      try {
+        const data = await response.json();
+        if ("exception" in data) {
+          setOrganizerError(true);
+          return;
+        }
+        const organizers = data.map(
+          (organizer: { name: string | { nl: string }; id: string }) => {
+            if (typeof organizer.name === "object" && "nl" in organizer.name) {
+              return { name: organizer.name.nl, id: organizer.id };
+            }
+            return organizer;
+          }
+        );
+        setOrganizerList(organizers);
+      } catch (error) {
+        setOrganizerError(true);
+        return;
+      }
+
+      if (organizerError) {
+        setOrganizerError(false);
+      }
```

Comment:
if error set error to false?

Reply:
It clears out any error from the past attempt if the logic managed to reach this point, since actual errors will exit out of the logic before it